Appsilon/rhino
# Add `build_js()` and `lint_js()`

### Changes
Closes #27, closes #29.

### How to test
1. Initialize a project.
2. Run `rhino::build_js()`. The `app/js/index.js` should be compiled into `app/static/js/app.min.js`.
3. Add `const answer = 42` to `app/js/index.js` and run `rhino::lint_js()`. The linter should complain.
